eclass/python-utils-r1.eclass

```diff
@@ -1321,6 +1321,15 @@ _python_check_occluded_packages() {
 # Specifies the number of jobs for parallel (pytest-xdist) test runs.
 # When unset, defaults to -j from MAKEOPTS, or the current nproc.
 
+# @ECLASS_VARIABLE: EPYTEST_FLAGS
+# @USER_VARIABLE
+# @DEFAULT_UNSET
+# @DESCRIPTION:
+# Additional flags to pass to pytest.  This is intended to be set
+# in the environment when debugging packages (options such as -x or -s
+# are useful here), rather than globally.  It must not be set
+# in ebuilds.
```

Comment:
If I understand correctly, we don't support passing here a string with spaces, let's say `-m "not arthur"`. In such case, maybe we should document it in the comment here.

Reply:
It's just like `CFLAGS` and other kinds of flags.